pkg/utils/miniojob/types.go

```diff
@@ -126,8 +119,8 @@ func (jobCommand *MinIOIntervalJobCommand) Success() bool {
 	return jobCommand.Succeeded
 }
 
-// CreateJob - create job
-func (jobCommand *MinIOIntervalJobCommand) CreateJob(ctx context.Context, k8sClient client.Client, jobCR *v1alpha1.MinIOJob) error {
+// createJob - create job
+func (jobCommand *MinIOIntervalJobCommand) createJob(ctx context.Context, k8sClient client.Client, jobCR *v1alpha1.MinIOJob, stsPort string) (objs []client.Object) {
```

Comment:
Please change `stsPort` to an integer. This will also affect some other functions.